### PR TITLE
Modify the User Guide for help, list, and exit features

### LINKED ISSUE #103: Update DG for Help, List, and Exit Features

### PR DESCRIPTION
**Pull request by Daphne789**
Update the existing User Guide for help, list, and exit features

Update the helpWindow image to show the expected output

Add the listResult image to show the expected output

### CODE REVIEW

**Comment by syamfarh** [comment]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -65,12 +65,14 @@ FApro seeks to improve the quality of life of financial advisors (FAs). It allow
 
 ### Viewing help: `help`
 
-Shows a help window that lists all the main commands with their respective details and displays the link for more detailed 
-user guide.
+Shows a message listing all the main commands and explaining how to access the user guide.
```
Comment (limzhenwy) [inline docs/UserGuide.md:68]:
```suggestion
Shows a message listing all the main commands and explains how to access the user guide.
```

Comment (limzhenwy) [reply docs/UserGuide.md:68]: minor grammer nitpick

**Review by limzhenwy — CHANGES_REQUESTED** [review]
LGTM. One very minor nit!

**Pull request merged**